src/odemis/gui/cont/tabs.py

```diff
@@ -1796,6 +1796,15 @@ def __init__(self, name, button, panel, main_frame, main_data):
             calibrations
         )
 
+        # Controller for acquistion settings panel
+        self._acq_settings_controller = settings.SettingsController(
+            panel.pnl_acq_settings,
+            "dwell time"
+        )
+        dt_conf = {"range": [1e-6, 40e-6]}  # Limit the values the user can set the dwell time to.
+        self._acq_settings_controller.add_setting_entry(
+            "dwell time", tab_data.main.multibeam.dwellTime, tab_data.main.multibeam, dt_conf)
```

Comment:
The first parameter is the name of the VA, which should be just the string "dwellTime". It will be converted automatically to something nice looking by the function.